server/cmd/dexcoin/go.sum

```diff
@@ -33,6 +33,7 @@ github.com/dchest/siphash v1.2.1/go.mod h1:q+IRvb2gOSrUnYoPqHiyHXS0FOBBOdl6tONBl
 github.com/decred/base58 v1.0.0/go.mod h1:LLY1p5e3g91byL/UO1eiZaYd+uRoVRarybgcoymu9Ks=
```

Comment:
These changes snuck in on you somehow. Please undo.

Reply:
Done.